src/context_store/ingestion/pipeline.py

```
@@ -291,23 +283,20 @@ async def _process_chunk_locked(
             project=str(project) if project else None,
         )
 
-        # ステップ5: 重複排除（類似度チェック）
-        dedup_result = await self._deduplicator.deduplicate(memory)
+        # ========================================================
+        # 重複排除（Deduplication）のみをロックで保護
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)